src/js/App/Routes/QuickstartsCatalogRoute.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { getUrl } from '../../utils';
+import QuickStartCatalog from '../QuickStart/QuickStartCatalog';
+
+const QuickstartCatalogRoute = () => {
+  const bundle = getUrl('bundle');
+  const disabled = useSelector(
+    ({
+      chrome: {
+        quickstarts: { disabled },
+      },
+    }) => disabled
+  );
+
+  useEffect(() => {}, [bundle]);
```

Comment:
Is this empty useEffect good for anything?

Reply:
Yes, there will be some additional logic once the backend is deployed.

Reply:
Can we add the logic once the backend is deployed? :smile:

Reply:
done